package.json

```diff
@@ -11,7 +11,9 @@
     "terminal-screenshot": "out/src/cli.js"
   },
   "scripts": {
-    "lint": "jest && prettier . --list-different && tsc"
+    "lint": "prettier . --list-different",
+    "build": "tsc",
```

Comment:
@OmarTawfik rename `compile` to `build` which is more common

Reply:
Sounds good.